app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,57 @@
+class PostsController < ApplicationController
+
+  
+ 
+  #to render a particular post(s).
+  def show
+    @post = Post.find(params[:id])
+  end
+  
+  #to return a view with an HTML form to create a new post
+  def new
+   @post = Post.new 
+  end
+
+  #actually do the creating of said post
```

Comment:
No - really just for my own understanding in the processes of writing the code - so will go ahead and delete them.